packages/react/src/themes/types.ts

```diff
@@ -128,11 +153,6 @@ export type State = ObjectOf<any>
 // ========================================================
 
 export interface SiteVariablesInput extends ObjectOf<any> {
```

Comment:
No typings regarding the colors inside the site variables - those should be theme dependent.